File: frontend/public/js/models/MESMaterialConsumption.js
export class MESMaterialConsumption {
  constructor({
    id,
    productionOrderNumber,
    batchCode,
    quantity,
    ingredientCode,
    lot,
    unitOfMeasurement,
    datetime,
    operator_ID,
    supplyMachine,
    timestamp,
    count,
    request,
    respone,
    status,
    status1,
  }) {
    this.id = id;
    this.productionOrderNumber = productionOrderNumber;
    this.batchCode = batchCode;
    this.quantity = quantity;
    this.ingredientCode = ingredientCode;
    this.lot = lot;
    this.unitOfMeasurement = unitOfMeasurement;
    this.datetime = datetime;
    this.operator_ID = operator_ID;
    this.supplyMachine = supplyMachine;
    this.timestamp = timestamp;
    this.count = count;
    this.request = request;
    this.respone = respone;
    this.status = status;
    this.status1 = status1;
  }
}

File: backend/Models/MesmaterialConsumption.cs
using System;
using System.Collections.Generic;

namespace apiserver.Models;

public partial class MesmaterialConsumption
{
    public int Id { get; set; }

    public string? ProductionOrderNumber { get; set; }

    public string? BatchCode { get; set; }

    public string? Quantity { get; set; }

    public string? IngredientCode { get; set; }

    public string? Lot { get; set; }

    public string? UnitOfMeasurement { get; set; }

    public DateTime? Datetime { get; set; }

    public string? OperatorId { get; set; }

    public string? SupplyMachine { get; set; }

    public DateTime? Timestamp { get; set; }

    public int? Count { get; set; }

    public string? Request { get; set; }

    public string? Respone { get; set; }

    public string? Status { get; set; }

    public string? Status1 { get; set; }
}
